cdap-app-fabric/src/main/java/org/apache/twill/yarn/YarnTwillController.java

```diff
@@ -117,7 +117,7 @@ protected void doStartUp() {
 
       YarnApplicationReport report = processController.getReport();
       ApplicationId appId = report.getApplicationId();
-      LOG.debug("Application {} with id {} submitted", appName, appId);
+      LOG.info("Application {} with id {} submitted", appName, appId);
```

Comment:
Right. I've made this change in Twill also, so when this class gets removed from CDAP, the change is still in effect: https://github.com/apache/twill/pull/17.
Users should be able to change the logback to INFO, and then see this.
Currently, they have to set it to DEBUG to see it, and too many log message may come with it set to DEBUG.